statsmodels/graphics/functional.py

```diff
@@ -472,8 +472,8 @@ def _band_quantiles(band, use_brute=use_brute, seed=seed):
     return fig, hdr_res
 
 
-def fboxplot(data, xdata=None, labels=None, depth=None, method='MBD',
-             wfactor=1.5, ax=None, plot_opts=None):
+def fbplot(data, xdata=None, labels=None, depth=None, method='MBD',
```

Comment:
We can't rename functions without a deprecation cycle.

Reply:
Thx,  got it. Then, can I upload a new version of the functional boxplot, instead of revising the old one?

Reply:
We could create an alias to the name and start a deprecation.

Reply:
No no need to add a very similar function.  There should be only 1 (the better version, which is this one being worked on).

Reply:
Alright, thx, I got it!